doc/install.md

```diff
@@ -105,6 +105,66 @@ If you decided to use the system rocksdb (recommended if the target OS supports
 $ sudo apt install librocksdb-dev:arm64
 ```
 
+#### Preparing for cross compilation on a different (Debian-based) OS distribution/version
+
+If your build system runs on a different OS distribution and/or release than the target system electrs is going to run on, you may run into `GLIBC` version issues like:
```

Comment:
Cool! Maybe this should contain a note that unless one runs into libc issue, avoiding debootstrap is faster?

Reply:
Done